### ISSUE TITLE
Fee presentation in the UI and in the confirmation modal

### ISSUE THREAD

**Issue by MareenG** [gnosis/gp-swap-ui#396]
In the new release the fees are deducted  from and displayed in the **TO** token 
<img width="390" alt="Bildschirmfoto 2021-04-08 um 13 07 03" src="https://user-images.githubusercontent.com/34510341/114017950-f9d04d00-986c-11eb-97e2-4d8d4669b83f.png">
<img width="409" alt="Bildschirmfoto 2021-04-08 um 13 08 15" src="https://user-images.githubusercontent.com/34510341/114017944-f8068980-986c-11eb-9bf3-ce3aa08d4179.png">

This is clearly a better user experience than in the prior version.

However when signing a transaction the fees are deducted  from and displayed in the **FROM** token
<img width="352" alt="Bildschirmfoto 2021-04-08 um 13 07 31" src="https://user-images.githubusercontent.com/34510341/114018059-1cfafc80-986d-11eb-9bdd-a341a5cca481.png">

I think this is quite confusing for a user that the fee is now in the FROM token.

**Comment by anxolin** [comment]
I agree @MareenG, thanks for reporting this, although I'm afraid this one was a deliberate decision.

The tooltip tries to be kind of a "invoice details" explaining the user the number he is seeing in the inputs. This number is obtain using the prices we show the user, and applying the fee. The issue is that for sell orders, it would be harder to explain this number if we use different units (receive units for the received tokens before fees, and the fees in sell tokens). 

From a user's point of view, our assumption is that it doesn't matter if the user pays in sell or buy token. It would be if we take the tokens after or before converting.

This is just an iteration, so next thing will be to see what users say about it. I think you have QA googles on, and same as DEV you notice this, but I'm not sure if people will actually complain about this. Maybe yes, but I'd like to find out. So for now we are going for making the "invoice details more clear"

**Comment by MareenG** [comment]
Agree that it makes sense to wait for some user feedback and maybe people won't notice this. 
I think one easy fix might to to give the option to additionally show the fees in sell token as well. A bit similar as we do for the price which can be displayed in Token A per Token B or Token B per Token A. 
But this could be also evenmore confusing if there is no further context :see_no_evil:
